### PR TITLE
Add more system tools and descriptions

### PR DESCRIPTION
**Pull request by vvasuki**

### CODE REVIEW

**Comment by helmuthdu** [comment]
Hi and thanks for you contribution, can you just add those changes into de lilo.automode file?

**Comment by vvasuki** [comment]
Thanks - fixed, please take another look.

File: lilo.automode [hunk 1]
```diff
@@ -183,19 +183,22 @@
...
+  #10) Plex
+  #11) UFW
+  #12) Unified Remote
+  #13) Virtualbox
+  #14) Wine
+  #15) netdata
+  #16) nload
+  #17) Assorted Graphical log viewers
+  #18) seahorse Password keyring management GUI
   SYSTEMTOOLS_OPTIONS="10"
```
Comment (Rfvgyhn) [inline lilo.automode]: With webmin pushing the order by one, this will now install plex by default instead of UFW.

Comment (vvasuki) [reply lilo.automode]: Thanks for catching that- fixed now.

**Comment by helmuthdu** [comment]
Hi,
Thank you for all your contributions and sorry for the delay, I was extremely busy these days.

**Pull request merged**